File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/user/PartnerApiTokenResultDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.user;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkMessageAndResultDto;

/**
 * APIパートナー長期トークン発行Dto
 */
public class PartnerApiTokenResultDto extends FrameworkMessageAndResultDto { // NOPMD DataClass

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** トークン */
    private String token = INIT_STRING;

    /**
     * トークンを取得する
     * 
     * @return トークン
     */
    public String getToken() {
        return token;
    }

    /**
     * トークンを設定する
     * 
     * @param token トークン
     */
    public void setToken(final String token) {
        this.token = token;
    }

}

File: app/front/src/components/main/dto/user/partnerApiTokenResultDto.ts
import { FrameworkMessageAndResultDto, type FrameworkMessageAndResultDtoInterface } from "seijishikin-jp-normalize_common-tool";

interface PartnerApiTokenResultDtoInterface extends FrameworkMessageAndResultDtoInterface {

    /**　トークン */
    token: string;
}

class PartnerApiTokenResultDto extends FrameworkMessageAndResultDto implements PartnerApiTokenResultDtoInterface {

    /**　トークン */
    token: string;

    constructor() {
        super();
        this.token = "";
    }
}

export { type PartnerApiTokenResultDtoInterface, PartnerApiTokenResultDto }